### PR TITLE
[mod_tags_similar]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_tags_similar
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_tags_similar/mod_tags_similar.xml [hunk 1]
```diff
@@ -25,69 +25,72 @@
...
+					<option value="all">MOD_TAGS_SIMILAR_FIELD_ALL</option>
+					<option value="any">MOD_TAGS_SIMILAR_FIELD_ONE</option>
+					<option value="half">MOD_TAGS_SIMILAR_FIELD_HALF</option>
 				</field>
+
 				<field
 					name="ordering"
 					type="list"
 					default="count"
 					label="MOD_TAGS_SIMILAR_FIELD_ORDERING_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_tags_similar/mod_tags_similar.xml]: shouldn't default be moved down?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 01b2dffb706b6cc48e2e0e94044057942f27fc21<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10595">issues.joomla.org/joomla-cms/10595</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 01b2dffb706b6cc48e2e0e94044057942f27fc21<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10595">issues.joomla.org/joomla-cms/10595</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10595">issues.joomla.org/joomla-cms/10595</a>.</sub>

**Pull request closed without merging**